rllib/evaluation/env_runner_v2.py

```diff
@@ -79,6 +99,28 @@ def get(self):
             "mean_env_render_ms": self.env_render_time * factor,
         }
 
+    def _get_ema(self):
+        # In EMA mode, stats are already (exponentially) averaged,
+        # hence we only need to do the sec -> ms conversion here.
+        return {
+            # Raw observation preprocessing.
+            "mean_raw_obs_processing_ms": self.raw_obs_processing_time * 1000,
```

Comment:
define 1000 in a MS_TO_SEC_FACTOR variable

Reply:
done